opensrp-chw-hf/src/main/res/values/strings.xml

```diff
@@ -28,4 +28,5 @@
     <string name="error_opening_profile">Error opening client\'s profile</string>
     <string name="family">%s Family</string>
     <string name="created">Created: %s</string>
+    <string name="mapbox_access_token">pk.eyJ1IjoiYm9yZXNoYS1hZnlhIiwiYSI6ImNrYTZzODVudDBhMWIycm1vbzF6dHc4YzgifQ.EPhVvPOIeQEkKwAyyU7fIg</string>
```

Comment:
Revert this and remove this key from the public repository. I am going to have to revoke this key on Mapbox.

Alternatively, create another PR to make this available from the local.properties file while I revoke and regenerate a new one